string value_name
uint8 value_amount
---
string status